src/decanter/core/core_api/predict_input.py

```diff
@@ -27,7 +28,7 @@ class PredictInput:
 
     """
     def __init__(
-            self, data, experiment, callback=None,
+            self, data, experiment, select_model='best', callback=None, 
```

Comment:
I thought this parameter is weird. How about make `predict_method` and `predict_opt` as parameters?
e.g. predict_method = predict_method.CV_BEST, predict_opt = None or predict_method = predict_method.MODEL_ID, predict_opt = 00aabbcc11

Reply:
After updated the parameters, please update the docstring.